### DISCUSSION TITLE
Putlistingitem to add an offer to an existing ASIN

### DISCUSSION THREAD

**Discussion by astanley44** [abuzuhri/Amazon-SP-API-CSharp#774] [General]
I am trying to add offers to existing ASINS using putlistingitem.
I have LISTING_OFFER_ONLY and productype as PRODUCT
It works in sandbox but I get this error on live

Creating products is not supported with the ‘PRODUCT’ Amazon product type. Either specify a specific Amazon product type or specify an offer-only requirements set.

I assume its either LISTING_OFFER_ONLY  not working or it's no longer allowed for the PRODUCT producttype and I need to use the correct producttype for each product and associated required attributes, which is an absolute nightmare.

Can anybody confirm either way?

**Comment by nitsujT** [comment]
Would you mind sharing the code you have for doing this? as I'm trying to add offers to ASINs but keep getting errors returned and think its due to how I'm sending over the ASIN and sku data.
